### PR TITLE
WTrialBalance ORG_ID should not be mandatory by default

### PR DESCRIPTION
**Pull request by SusanneCalderon**
Fixes https://github.com/adempiere/adempiere/issues/1688

This pull request replaces the pullrequest https://github.com/adempiere/adempiere/pull/1780, which contains by error the file _adempiere.iml_

### CODE REVIEW

**Comment by SusanneCalderon** [comment]
Hello, I closed the erroneous pullrequest https://github.com/adempiere/adempiere/pull/1780.

File: zkwebui/WEB-INF/src/org/adempiere/webui/apps/form/WTrialBalance.java [hunk 1]
```diff
@@ -693,10 +717,24 @@ private void setPeriodToStyle()
 	 * Set style on organization Label
 	 */
 	private void setOrgStyle()
-	{
-		if (fieldOrg.getValue() == null)
+	{MAcctSchema acctSchema = MAcctSchema.getClientAcctSchema(Env.getCtx(), Env.getAD_Client_ID(Env.getCtx()))[0];
```
Comment (yamelsenih) [inline zkwebui/WEB-INF/src/org/adempiere/webui/apps/form/WTrialBalance.java]: Add a new line here please

File: zkwebui/WEB-INF/src/org/adempiere/webui/apps/form/WTrialBalance.java [hunk 2]
```diff
@@ -693,10 +717,24 @@ private void setPeriodToStyle()
 	 * Set style on organization Label
 	 */
 	private void setOrgStyle()
-	{
-		if (fieldOrg.getValue() == null)
+	{MAcctSchema acctSchema = MAcctSchema.getClientAcctSchema(Env.getCtx(), Env.getAD_Client_ID(Env.getCtx()))[0];
+		if (acctSchema == null)
+		{
+			StringBuffer errorMessage = new StringBuffer();
+				errorMessage.append("@No Acctschema@");
```
Comment (yamelsenih) [inline zkwebui/WEB-INF/src/org/adempiere/webui/apps/form/WTrialBalance.java]: Use 
<pre>
errorMessage.append("@C_AcctSchema_ID@ @NotFound@");
</pre>

File: zkwebui/WEB-INF/src/org/adempiere/webui/apps/form/WTrialBalance.java [hunk 3]
```diff
@@ -323,11 +325,33 @@ private void loadTable()
...
+			errorMessage.append("@C_PeriodTo_ID@");
+		}
+		if (orgMandatory && m_AD_Org_ID == 0) {
+			if(errorMessage.length() > 0) {
+				errorMessage.append(Env.NL);
+			}
+			errorMessage.append("@AD_Org_ID@");
+		}
+		if(errorMessage.length() > 0) {
+			FDialog.error(form.getWindowNo(), "FillMandatory", errorMessage.toString());
```
Comment (yamelsenih) [inline zkwebui/WEB-INF/src/org/adempiere/webui/apps/form/WTrialBalance.java]: Change for:
<pre>
FDialog.error(form.getWindowNo(), "FillMandatory", Msg.parseTranslation(Env.getCtx(), errorMessage.toString()));
</pre>

**Pull request merged**